cmd/crowdsec-cli/dashboard.go

```diff
@@ -109,6 +112,22 @@ cscli dashboard setup -l 0.0.0.0 -p 443 --password <password>
 				}
 			}
 			var answer bool
+			if valid, err := checkSystemMemory(); err == nil && !valid {
+				if !forceYes {
+					prompt := &survey.Confirm{
+						Message: "Metabase requires 1-2GB of RAM, your system is below this requirement continue ?",
+						Default: true,
+					}
+					if err := survey.AskOne(prompt, &answer); err != nil {
+						log.Fatalf("unable to ask about RAM check: %s", err)
```

Comment:
we might want to warn/error on this but not fatal : if for any reason the check errors, we're preventing the user from setting up the dashboard